src/cascadia/TerminalControl/TermControl.h

```diff
@@ -161,8 +168,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         void _BackgroundColorChanged(const uint32_t color);
         bool _InitializeTerminal();
         void _UpdateFont();
+        void _KeyDownHandler(Windows::Foundation::IInspectable const& /*sender*/, Windows::UI::Xaml::Input::KeyRoutedEventArgs const& e);
         void _SetFontSize(int fontSize);
-        void _KeyDownHandler(Windows::Foundation::IInspectable const& sender, Windows::UI::Xaml::Input::KeyRoutedEventArgs const& e);

```

Comment:
I remember that I did this because an unused input parameter will cause some build checks to fail (on gitHub), I wonder if you saw this happen before ?

Reply:
undone